app/src/main/java/org/listenbrainz/android/ui/screens/feed/FollowListensPagingSource.kt

```diff
@@ -17,11 +17,8 @@ class FollowListensPagingSource(
     private val ioDispatcher: CoroutineDispatcher
 ): PagingSource<Int, FeedUiEventItem>() {
     
-    override fun getRefreshKey(state: PagingState<Int, FeedUiEventItem>): Int? {
-        return if ((state.anchorPosition ?: 0) < 10)
-            null
-        else
-            state.lastItemOrNull()?.event?.created
+    override fun getRefreshKey(state: PagingState<Int, FeedUiEventItem>): Int {
+        return (System.currentTimeMillis()/1000).toInt()
```

Comment:
This doesn't seem like the best solution.

Reply:
`The getRefreshKey method of a PagingSource is used to determine which page key should be used if a refresh is triggered (i.e., when the data gets invalidated and needs to be loaded from the start).` How is the solution relevant?

Reply:
it is. This is just the key that will be given to pager when we do swipe to refresh. Also, Server takes time in milliseconds

Reply:
Every time we do swipe to refresh, this function will be called and all the paging data reset.

Reply:
Oh so the server takes the nearest timestamp and returns the feed accordingly? I was of the opinion it would take in an exact timestamp

Reply:
Okay the `max_ts` here would be this then

Reply:
It just does ts <= max_ts on the backend

Reply:
Cool

Reply:
Actually, providing null would also do the job since null means the server will take current time automatically.

Reply:
Ouu 10 seconds too late.